sympy/functions/combinatorial/factorials.py

```diff
@@ -474,22 +472,22 @@ def _eval_rewrite_as_gamma(self, n):
 
 
 class RisingFactorial(CombinatorialFunction):
-    """
+    r"""
     Rising factorial (also called Pochhammer symbol) is a double valued
     function arising in concrete mathematics, hypergeometric functions
     and series expansions. It is defined by:
 
-                rf(x, k) = x * (x + 1) * ... * (x + k - 1)
+    .. math:: rf(x,k) = x*(x+1)* \cdots *(x+k-1)
```

Comment:
The `*`s could probably be omitted.

Reply:
I was thinking of omitting this, but the definition of `rf` below for Poly instance seems to be using it (which is required to differentiate functional braces and juxtaposition).

Reply:
Ok. That is logical.

Reply:
Could `\cdot` be used?

Reply:
Yes, it could be `rf(x,k) = x \cdot (x+1) \cdots (x+k-1)` 
and `rf(x,k) = x(y) \cdot x(y+1) \cdots x(y+k-1)`.